client/src/components/GraphViewerComponent/GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent.js

```diff
@@ -474,7 +474,7 @@ export class GraphViewerCytoscapeComponent extends React.Component {
     });
   }
 
-  doLayout() {
+  doLayout(nodeColorUpdateOnly = false) {
```

Comment:
I think pulling out the node color logic into a separate method is probably clearer here imho. I.e. and call that method instead rather than passing a flag here.

Reply:
Done